zcash_keys/src/address.rs

```diff
@@ -199,6 +222,7 @@ impl From<UnifiedAddress> for Address {
 impl TryFromRawAddress for Address {
     type Error = &'static str;
 
+    #[cfg(feature = "sapling")]
```

Comment:
Yay, this is exactly the kind of use case that the `zcash_address` crate conversion traits are designed to handle 😄